docs/elevate/ELevate-NG-testing-guide.md

```diff
@@ -119,7 +121,7 @@ When successfully migrated to AlmaLinux 8 OS, consider performing these steps to
     leapp-data-almalinux-0.2-6.el7.noarch
     leapp-0.16.0-2.el7.noarch
    ```
-   As mentioned above, consider removing these packages or upgrading them manually to proceed with migration to AlmaLinux 9.
+   As mentioned above, consider removing these packages or upgrading them manually to proceed with the update to AlmaLinux 9.
```

Comment:
Update or upgrade?  I do like the differentiation here so long as you weren't trying to be consistent throughout.

Reply:
upgrade, for sure.